src/test/ui/async-await/issue-69446-fnmut-capture.stderr

```diff
@@ -0,0 +1,19 @@
+error: captured variable cannot escape `FnMut` closure body
+  --> $DIR/issue-69446-fnmut-capture.rs:19:17
+   |
+LL |       let mut x = Foo;
+   |           ----- variable defined here
+LL |       bar(move || async {
+   |  _______________-_^
+   | |               |
+   | |               inferred to be a `FnMut` closure
+LL | |         x.foo();
+   | |         - variable captured here
+LL | |     });
+   | |_____^ returns an `async` block that contains a reference to a captured variable, which then escapes the closure body
```

Comment:
woah. These multi-line spans are always kind of intense :/ not sure what can be done about that particular problem just now though

Reply:
Yeah, it does look kind of visually overwhelming. I think this would be a case where we would greatly benefit from some kind of 'interactive mode' for diagnostics. For example, the 'inferred to be a `FnMut` closure' message is potentially useful, but doesn't necessarily need to be shown by default.

Of course, that isn't compatible with plain text output - you'd need integration with a browser/IDE.